brambling/models.py

```diff
@@ -1098,6 +1098,10 @@ def get_eventhousing(self):
     def has_dwolla_payments(self):
         return self.transactions.filter(method=Transaction.DWOLLA).exists()
 
+    def awaiting_check_confirmation(self):
```

Comment:
I feel like I started writing a better version of this, but it got lost in the mists of time, I'll see if I can find where I left off.